src/scripts/components/DropdownMultiSelectAdvanced.jsx

```diff
@@ -21,6 +21,13 @@ class DropdownMultiSelectAdvanced extends React.Component {
     isViewAllItems: true,
   };
 
+  static getDerivedStateFromProps(nextProps) {
+    if (nextProps.selectedItemsIds.length === 0) {
```

Comment:
```suggestion
    if (nextProps && nextProps.selectedItemsIds && nextProps.selectedItemsIds.length === 0) {
```

Reply:
Actually, This selectedItemsIds prop is a required prop that's why I didn't check this condition here, becoz it will be always there.